src/components/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import Banner from './Banner';
+import { BannerVariants } from './Banner.enums';
+import { Text } from '../typography';
+
+export default {
+  title: 'components/Banner',
+  component: Banner,
+  decorators: [
+    (storyFn) => (
+      <div style={{ padding: '25px', marginBottom: '25px' }}>{storyFn()}</div>
+    ),
+  ],
+} as Meta;
+
+export const InfoBanner: Story = () => (
+  <Banner
+    actions={[
+      {
+        label: 'Action A',
+        name: 'Action A',
+        onClick: action('click-action-a'),
+      },
+      {
+        label: 'Action B',
+        name: 'Action B',
+        onClick: action('click-action-b'),
+      },
+    ]}
+    variant={BannerVariants.info}
+    onClose={action('close-banner')}
+  >
+    <Text size="md">Info Banner</Text>
+  </Banner>
+);
+
+export const LongInfoBanner: Story = () => (
+  <Banner
+    actions={[
+      {
+        label: 'Action A',
+        name: 'Action A',
+        onClick: action('click-action-a'),
+      },
+      {
+        label: 'Action B',
+        name: 'Action B',
+        onClick: action('click-action-b'),
+      },
+    ]}
+    variant={BannerVariants.info}
+    onClose={action('close-banner')}
+  >
+    <Text size="md">
```

Comment:
@PrincAm I'm thinking if we should give people this amount of freedom. If it wouldn't be better to define the text size inside the component itself and then they don't need to think about wrapping children in `Text` component.

Reply:
Yep, it would be easier for user. I will move `Text` inside